Cargo.toml

```diff
@@ -66,3 +66,8 @@ strip = true
 opt-level = 3
 lto = true
 codegen-units = 1
+
+[profile.release.package."*"]
```

Comment:
What does this do? Dont we only have one package in our repo anyways? Or does it apply to our dependencies? (Although I would expect that the other config block above this one covers everything)

Reply:
Think it would cool to add that :3